docs/performance/mobile-performance-tuning.md

```diff
@@ -6,6 +6,8 @@ redirect_from: /docs/how-to/mobile-performance-tuning
 ---
 {::options toc_levels="2" /}
 
+**The information on this page applies to ONNX Runtime version 1.11 and later. See the documentation for older versions [here](./mobile-performance-tuning-1.10-and-earlier.md).**
+
 # ONNX Runtime Mobile Performance Tuning
```

Comment:
nit: with the updates to the conversion script this info is really an explanation rather than 'tuning'. Should it maybe move under the 'reference' info for Mobile?

Reply:
updated the organization and content a bit. didn't find a good way to integrate the original info.